src/Tools/BuildBoss/CompilerNuGetCheckerUtil.cs

```diff
@@ -138,79 +124,6 @@ private bool CheckCoreClr(TextWriter textWriter)
                         dllRelativeNames);
         }
 
-        private bool CheckPortableFacades(TextWriter textWriter)
```

Comment:
Where did this verification move to?

Reply:
It's correct by construction.

Reply:
I mean it doesn't seem to make sense to have two lists - one that lists what we want to include and the other that lists what we don't want to include and check that the first doesn't contain the latter.

Reply:
Besides, soon the list of facades will shrink to a few items.

Reply:
I mean to zero. We are not gonna need to build and insert PortableFacades.vsix.